benches/coprocessor_executors/mod.rs

```diff
@@ -11,14 +11,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#![allow(clippy::blacklisted_name)]
```

Comment:
We may frequently use foos and bars elsewhere in the project. I think it's better to be a global name until https://github.com/rust-lang/rust-clippy/issues/2198 is resolved.